common/test/acceptance/tests/lms/test_lms_courseware.py

```diff
@@ -475,6 +484,96 @@ def test_navigation_buttons(self):
         self.courseware_page.click_previous_button_on_bottom()
         self.assert_navigation_state('Test Section 1', 'Test Subsection 1,1', 2, next_enabled=True, prev_enabled=True)
 
+        sequence_ui_events = self.wait_for_events(event_filter=filter_sequence_ui_event)
```

Comment:
You may want to consider adding a timeout to this or a "number_of_matches" parameter. Right now it will return as soon as *any* matching event is detected. Given that they are fired async from the browser, I would recommend a short timeout.